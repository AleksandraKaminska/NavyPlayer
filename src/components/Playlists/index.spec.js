import React from 'react'
import { Playlists } from './index'
import renderer from 'react-test-renderer'

describe('Playlists', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Playlists top={{}} />)
    expect(tree).toMatchSnapshot()
  })
})
