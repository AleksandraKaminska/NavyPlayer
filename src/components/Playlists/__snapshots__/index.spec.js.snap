// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Playlists renders correctly 1`] = `null`;
